src/Tempest/Http/src/Routing/Construction/RouteTreeNode.php

```diff
@@ -12,12 +12,12 @@
 final class RouteTreeNode
 {
     /** @var array<string, RouteTreeNode> */
-    private array $staticPaths = [];
+    public array $staticPaths = [];
```

Comment:
My guess is that these props can't be readonly?

Reply:
Correct these are mutated curing route construction. see `findOrCreateNodeForSegment` and `setTargetRoute` in this file